plugins/poll/spec/controllers/posts_controller_spec.rb

```diff
@@ -130,7 +130,6 @@
       expect(response.status).to eq(200)
       json = response.parsed_body
       expect(json["cooked"]).to match("data-poll-")
-      expect(json["cooked"]).to include("&lt;script&gt;")
```

Comment:
Does it no longer include the escaped script tag?

Reply:
Yes it does not, due to the same reason `@flavorjones` mentioned above. The cooked value is now

```
<div class="poll" data-poll-status="open" data-poll-name="<script>alert('xss')</script>"> 
...
```

where we concluded above that `<` in attributes are actually deemed ok / a valid attribute string.